src/lua/api-gateway/util/logger.lua

```diff
@@ -67,6 +67,23 @@ local function _decorateLogger()
     end
 end
 
+logger = {}
+
+function logger.error(...)
+    _decorateLogger()
+    ngx.log(ngx.ERR, ...)
+end
+
+function logger.debug(...)
+    _decorateLogger()
+    ngx.log(ngx.DEBUG, ...)
+end
+
+function logger.info(...)
+    _decorateLogger()
+    ngx.log(ngx.INFO, ...)
+end
```

Comment:
why not expose all the functions in the return bellow. and in factory require it above all other modules globally like so: logger = require ("....logger")